sale_quotation_capable_vendor_rfq/models/sale_order.py

```diff
@@ -0,0 +1,167 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from collections import defaultdict
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    rfq_ids = fields.One2many(
+        "purchase.order",
+        "sale_quotation_id",
+        string="RFQs",
+        help="RFQs created from this sale order",
+    )
+    rfq_count = fields.Integer(
+        string="RFQ count",
+        compute="_compute_rfq_count",
+        help="Number of RFQ created from this sale order",
+    )
+
+    @api.depends("rfq_ids")
+    def _compute_rfq_count(self):
+        """
+        Compute the number of RFQs created from this sale order
+        """
+        for order in self:
+            order.rfq_count = len(order.rfq_ids)
+
+    def action_view_rfq(self):
+        """
+        View the purchase orders created from this sale order
+        """
+        self.ensure_one()
+        action = self.env.ref("purchase.purchase_rfq").sudo().read()[0]
```

Comment:
```suggestion
        action = self.env["ir.actions.actions"]._for_xml_id("purchase.purchase_rfq")
```
This seems to be more align with new Odoo code-base these days